### DISCUSSION TITLE
Review Assess functions and identify those that contain a lot of overlapping code.

### DISCUSSION THREAD

**Discussion by mattroumaya** [Gilead-Public/gsm#1515] [General]
- @Zhongkai-Wang and @mattroumaya to review Assess functions and identify those that contain a lot of overlapping code.
- Goal would be to use `Assess_Binary()` within several KRIs
  - e.g., within `PD_Assess()` and `PFS_Assess()`, might both call `Assess_Binary()`.
  - Target `v1.9.2` or `v1.10.0`

_Originally posted by @mattroumaya in https://github.com/Gilead-BioStats/gsm/discussions/1480_

**Comment by jwildfire** [comment]
@mattroumaya @Zhongkai-Wang Any progress on this? I'd like to implement in the next major release.
